src/rlang/vec.c

```diff
@@ -226,3 +226,41 @@ void r_vec_poke_range(r_obj* x, r_ssize offset,
                       r_obj* y, r_ssize from, r_ssize to) {
   r_vec_poke_n(x, offset, y, from, to - from + 1);
 }
+
+bool _r_is_finite(r_obj* x) {
```

Comment:
Is `_r` meaning that its extremely internal?

Reply:
yes it means it's not part of the API. I had to pull them from `internal/`, but the interface of these functions still need a bit of review.